### PR TITLE
Throw informative error messages

### PR DESCRIPTION
**Pull request by gkushang**
Error thrown from the module are not informative enough, for e.g. below error message does not say much what could be the error during plugin setup

```
nemoPluginSetupError: Nemo plugin threw error during setup
    at pluginReg (/jobvolume/........)
```

This fix improves the error messages,

```
nemoPluginSetupError: Nemo plugin threw error during setup. 
Error: Sauce labs username and/or accessKey is undefined. Please specify through config.
    at pluginReg (/Users....
```

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -50,8 +50,9 @@ function Nemo(_basedir, _configOverride, _cb) {
 
   //settle arguments
   if (arguments.length === 0) {
-    error('Nemo constructor needs at least a callback');
-    var noCallbackError = new Error('Nemo constructor needs at least a callback');
+    var ERROR_MESSAGE = 'Nemo constructor needs at least a callback';
```
Comment (grawk) [inline index.js]: would prefer camelCasing here. uppercase/titlecase should be reserved for named constants.

**Comment by grawk** [comment]
no issues with this PR aside from changing TITLE_CASE vars to camelCase.

File: index.js (hunk 1 above)
Comment (gkushang) [inline index.js]: updated to camelCase

**Comment by nikulkarni** [comment]
Looks good 👍

**Pull request merged**